gprofiler/metadata/application_identifiers.py

```diff
@@ -287,6 +303,7 @@ def get_app_id(self, process: Process) -> Optional[str]:
 }
 
 if is_linux():
+    _IDENTIFIERS_MAP["node"] = [_NodeModuleApplicationIdentifier()]
```

Comment:
Can be under the generic one (not only under `is_linux()`, that is, next to the python and others)